api-tests/specs/practitioner-service/linkperson.js

```diff
@@ -0,0 +1,125 @@
+/*
+Link person class file that has scenarios to run link person tests
+based on Sameera Purini's createperson class file
+
+Author - Ben Deinde
+*/
+
+const requests = require('../../data/practitioner-service/linkperson')
+const conf = require('../../config/conf')
+const utils = require('../../common/utils')
+const fetch = require("node-fetch");
+const { expect } = require('chai');
+const practitionerEndpointUri = '/api/practitioner'
+const linkpersonEndpointUri = '/api/practitioner/{personId}/link'
+
+
+describe('As a user with Update Person permission I want to link a Person to an existing User So that they have the Persons roles when they login to the system', function () {
+
+    it('Given a Person exists who is not linked to a User AND a User exists that is not linked to a Person When I submit an API request to link a Person who is NOT linked to a User to a User who is not linked to a Person in this trial Then the Person and User are linked and I receive a success acknowledgement ', async () => {
```

Comment:
The Given, When, then seems a little confusing...would this be better..
Given 
Person A exists who is not linked to a User AND User B exists that is not linked to a Person 
When I submit an API request 
to link a Person A to User B 
Then 
Person A and User B are linked and I receive a success acknowledgement